tests/range/range_api.cpp

```diff
@@ -254,6 +268,27 @@ class TEST_NAME : public util::test_base {
         test2d(log, range_2d_g, range_2d_l, my_queue);
         test_range<3> test3d;
         test3d(log, range_3d_g, range_3d_l, my_queue);
+#ifdef SYCL_CTS_COMPILING_WITH_COMPUTECPP
+        WARN(
+            "ComputeCpp does not implement unary minus operation. "
+            "Skipping the test for this operation.");
+#endif
+
+// The tests are using unnamed lambda
+#if SYCL_CTS_ENABLE_FULL_CONFORMANCE && \
+    !defined(SYCL_CTS_COMPILING_WITH_COMPUTECPP)
+        // 32 bits, it's trivial. Sanity test.
+        test_launch_kernel_1d_range(std::numeric_limits<uint32_t>::max(),
+                                    my_queue);
+
+        // Prime number bigger than UINT32_MAX
+        test_launch_kernel_1d_range(4294967311L, my_queue);
+
+        // Most GPU hardware have limitation to 32bits * 1024 for their native
+        // API so let's try more than that
+        test_launch_kernel_1d_range(
+            std::numeric_limits<uint32_t>::max() * 2024L, my_queue);
```

Comment:
What is `std::numeric_limits<uint32_t>::max() * 2024L` on my Vax11/785 with its FP164 accelerator?

Reply:
:( This is a valid point. Those magic numbers are always ugly. One should use interopts, and for each backend, compute the maximum number of work items and then add `+1`. 

In the beginning, I was planning to use `std::numeric_limits<size_t>` but the kernel will be quite long to submit. As a tradeoff, I can replace `2024` with `max_work_item_sizes*2`? On a few hardware I tests `max_work_item_sizes` return 1024 :)

Reply:
If I understand correctly we have two problem:
- uint32_t is optional so i cannot use it. Should I use `2^32-1` and call it a day or use `uint_least32_t` / `char32_t`?
- uint32_t * 20204L may overflow.  should I use `ull`?

Did I miss something else?

Reply:
`std::uint32_t` is probably here in the SYCL world (and I prefer the `std::` version to the C one, by the way :-) ), so just use it.

So the problem is the second point. Thus yes we need `ULL` (I prefer the uppercase version to avoid code obfuscation like `11ull` vs `11ULL`).